package.json

```diff
@@ -47,23 +50,25 @@
     }
   },
   "dependencies": {
-    "chalk": "~0.4.0",
-    "lodash": "~4.17.5"
+    "chalk": "^2.4.1",
+    "lodash": "^4.17.10"
   },
   "devDependencies": {
     "blanket": "^1.2.2",
-    "mocha": "^2.4.5",
+    "mocha": "^5.2.0",
     "node-mocks-http": "^1.5.1",
-    "promise": "^7.1.1",
-    "should": "^8.2.2",
+    "prettier-standard": "^8.0.1",
+    "should": "^13.2.3",
+    "standard": "^11.0.1",
     "travis-cov": "^0.2.5",
-    "winston": ">=1.x <3"
+    "winston": ">=3.x <4",
+    "winston-transport": "^4.2.0"
   },
   "peerDependencies": {
-    "winston": ">=1.x <3"
+    "winston": ">=3.x <4"
```

Comment:
does this mean express-winston longer supports Winston 2?

Reply:
That is correct. I outlined in the description why I think we should go that route:
>I think supporting both versions 2 and 3 of winston is going to cause too many headaches for it to be worth it. So just upgrade wholesale to version 3, and this will be a breaking change (and major version bump).

That is, we would release this change as express-winston 3.0.